default.nix

```
@@ -40,5 +40,7 @@ in {
 
   fly = pkgs.callPackage ./pkgs/fly.nix
     { source = sources.concourse; };
-
+  
+  tf  = pkgs.callPackage ./pkgs/tf.nix
+    { source = sources.tf; };
```

```suggestion
    { source = sources.tf; inherit terraform; };
```

You need to pass `terraform` of the toolbox, if not the standard `terraform` of nixpkgs will be used

---

For that to work, make the attribute set recursive on line 22: `in rec {`